project/Dependencies.scala

```diff
@@ -6,7 +6,7 @@ import org.portablescala.sbtplatformdeps.PlatformDepsPlugin.autoImport._
 
 object Dependencies {
   val metaconfigV = "0.9.10"
-  val scalametaV  = "4.4.17"
+  val scalametaV  = "4.4.18"
```

Comment:
now i'd definitely recommend this upgrade to be done separately

Reply:
https://github.com/scalameta/scalafmt/pull/2517